config/webpack.default.config.js

```diff
@@ -169,9 +169,10 @@ function getConfig(opts) {
         {
           test: /\.(jpe?g|gif|png|svg|woff|woff2|eot|ttf|wav|mp3|ico|mp4)$/,
           loader: "file-loader",
-          query: {
+          options: {
             context: "./app/assets",
-            name: config.outputFileName("[ext]")
+            name: config.outputFileName("[ext]"),
+            esModule: false
```

Comment:
why is `esModule: false` needed?

Reply:
That is needed if you are moving file-loader to 6.x.x version.

Reply:
ok